documentation/faq/80-integrations.md

```diff
@@ -13,3 +13,9 @@ Put the code to query your database in [endpoints](../docs#routing-endpoints) -
 ### How do I use Axios?
 
 You probably don't need it. We'd generally recommend you just use `fetch` instead. If you insist, you're probably better off using the ESM drop-in replacement `redaxios` [until axios utilizes ESM](https://github.com/axios/axios/issues/1879). Finally, if you still want to use Axios itself, try putting it in `optimizeDeps.include`.
+
+### How do I use Babel, CoffeeScript, Less, PostCSS / SugarSS, Pug, scss/sass, Stylus, TypeScript, `global` styles, or replace?
```

Comment:
can you move this up so that it's below the "How do I setup library X?" question. they seem fairly related so it'd probably be nice to put them next to each other

Reply:
done